%% METHODS & RESULTS %%

\section{Methods/Results}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% 1-TRIAL DESIGN %%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{Clinical Trial Design and Data Collection}

Our cohort was comprised of 436 patients with moderate to severe RA who participated in the phase III GO-FURTHER clinical trial \cite{weinblatt_radiographic_2014,weinblatt_intravenous_2013}. Each patient was randomized to either treatment or placebo arms (2:1) at week 0 and were followed for 100 weeks. Patients in the treatment arm were given 2mg/kg of golimumab intravenously at week 0 and 4 and every eight weeks thereafter. Patients in the placebo arm received placebo injections at weeks 0, 4, and 12. If a patient qualified for the “early escape” protocol (<10\% improvement from baseline), they were given intravenous golimumab at weeks 16 and 20 and every 8 weeks thereafter. If they did not qualify for early escape, they were given placebo injections at weeks 16 and 20 and were treated with golimumab at weeks 24 and 28 and every 8 weeks thereafter. To maintain blindness in the study, patients in the treatment arm were given placebo injections at weeks 16 and 24. All patients were on a concurrent MTX regimen throughout the study, despite having failed MTX treatment previously. \textcolor{green}{Thus, any placebo response observed in the trial resulted from active placebo.} Several patients dropped out of the study before completion. Patients who dropped out before the fourth week after treatment with golimumab were removed because their was insufficient evidence to determine their treatment response profile (Table \ref{Summary_Table}). Disease state was monitored at 16 pre-determined time points over 100 weeks, during which CRP, SJC, and TJC were assessed and DAS was calculated. Because patients were initiated on the drug at different time periods, by virtue of the trial design, there were 4 time points after initial golimumab treatment during which all patients were assessed: 4 weeks after golimumab (WAG), 12 WAG, 20 WAG, and 28 WAG.

Whole blood was taken from each patient during the clinical trial and a portion of the sample was sent to the Beijing Genomic Institute (BGI) for whole genome sequencing. Raw 90-bp, paired-end read sequences were produced in an average of \textasciitilde5 read groups per sample using an Illumina HiSeq 2500 platform. The average depth of overage by mapped reads was \textasciitilde35x. The average genome coverage is 99.4\%. Variant calls were made using a group-calling implementation of GATK’s best practices pipeline and quality control protocols confirmed the reliability of the results \cite{mckenna_genome_2010} \textcolor{orange}{(REF: OUR PREVIOUS PAPER)}.

\begin{table}
\centering
\def\arraystretch{1.2}
\begin{tabular}{| C{4cm} | C{2.5cm} | C{2.5cm} | C{2.5cm} | }
\hline
 & Golimumab & Placebo -\newline Early Escape & Placebo +\newline Early Escape \\
\hline
Num. Patients & 287 & 99 & 50 \\
Num. Removed (\%) & 5 (0.02) & 10 (0.1) & 0 (0) \\
Female (\%) & 227 (0.8) & 64 (0.72) & 41 (0.82) \\
Age (SD) & 51.76 (11.93) & 52.43 (11.28) & 49.24 (11.84) \\
Disease Duration (SD) & 7.21 (6.86) & 7.3 (8.07) & 6.56 (6.06) \\
BMI (SD) & 27.22 (5.78) & 26.63 (5.24) & 26.73 (6.65) \\
Initial DAS (SD) & 5.97 (0.82) & 5.81 (1.01) & 5.93 (0.8) \\
RF Positive (\%) & 261 (0.93) & 81 (0.91) & 45 (0.9) \\
ACPA Positive (\%) & 258 (0.91) & 83 (0.93) & 48 (0.96) \\
RF \& ACPA Positive (\%) & 237 (0.84) & 76 (0.85) & 43 (0.86) \\
Num. Obs. (SD) & 15.32 (1.88) & 15.57 (1.22) & 15.76 (0.72) \\
Num. Obs. on Gol. (SD) & 14.32 (1.88) & 6.6 (1.17) & 8.76 (0.72) \\
\hline
\end{tabular}
\caption{\label{Summary_Table} Summary of Clinical Trial Patients and Arms.}
\end{table}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% 2-ASSOC ASSUMPTIONS %%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{Assumptions of Parametric Association Methods}

Many common association methods in population genetics rely on parametric statistical tests. Frequently, these tests, including linear regression, t-tests, and linear mixed models, assume that the residuals of the data are normally distributed and homoscedastic. When we test these assumptions on many commonly used response phenotypes, we find that they are often violated. (Fig \ref{FIG_2}A) shows distributions of the change in DAS, CRP levels, SJC, and TJC for patients 20 weeks after their initial golimumab treatment. When we regress the change in disease state against the initial disease state, a common covariate included in genetic association studies, we find that several of the response metrics violate the assumption that residuals are normally distributed (Fig \ref{FIG_2}B). After transforming the metrics in the same way they are transformed to calculate DAS (square root for SJC and TJC (rSJC; rTJC), log10 for CRP (lCRP) ), the data better conform to these assumptions (Fig \ref{FIG_2}C). In addition, the assumption homoscedasticity is violated when regressing the change in SJC and TJC against their respective initial values (Fig \ref{FIG_2}D).

%% FIGURE 2 - PARAMETRIC ASSUMPTIONS
\begin{figure}[h!]
  \centering
  \includegraphics[width=0.9\textwidth]{Figs/2_Full.png}
  \caption{ {\bf Parametric Assumptions} {\bf A)} Phenotype distributions for change in DAS (red), CRP (yellow), SJC (green), and TJC (blue) 20 weeks after initial golimumab treatment. Shapiro-Wilk test p-values for change in phenotypes 4, 12, 20, and 28 weeks after golimumab (WAG), and using the first and last measurements (FL). {\bf B)} Distributions of residuals vs initial disease state for phenotypes at 20WAG. Shapiro-Wilk test p-values for residuals. {\bf C)} Distributions of residuals vs initial disease state for transformed phenotypes at 20WAG. Shapiro-Wilk test p-values for residuals of transformed phenotypes. {\bf D)} Residuals vs initial disease state for transformed phenotypes at 20WAG. Breusch-Pagan test of homoscedasticity for transformed phenotypes. \textcolor{orange}{Move to Supplemental Figures?}}
  \label{FIG_2}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% 3-PLACEBO %%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{Placebo Response}

\textcolor{green}{Where patients on MTX at baseline? Were they ever taken off MTX before being enrolled in trial?}

When analyzing the longitudinal results of the GO-FURTHER trial, we identified a statistically significant reduction in all disease metrics when patients were treated with active placebo (Fig \ref{FIG_3}A). The magnitude of the placebo effect resulted in a 12.2\% and 10.4\% improvement of initial value for DAS and lCRP, while it resulted in a 21.3\% and 16.8\% improvement from baseline for rSJC and rTJC, respectively (Fig \ref{FIG_3}B). Similarly, the magnitude of the placebo effect relative to the drug effect was larger for rSJC and rTJC that for lCRP (Fig \ref{FIG_3}B; DAS=38.5\%, lCRP=26.4\%, rSJC=44.5\%; rTJC=44.9\%). These results indicate that the active placebo had a larger effect on the swollen and tender joint counts than on the lCRP levels. As expected, the placebo effect size for DAS falls in the middle, since it is a composite score that includes lCRP, rSJC, and rTJC. This suggests that a study of drug response in RA using rSJC and rTJC is more likely to be confounded by placebo responders than those using lCRP or the composite DAS score.

%% FIGURE 3 - PLACEBO EFFECT
\begin{figure}[h!]
  \centering
  \includegraphics[width=0.9\textwidth]{Figs/3_Full.png}
  \caption{ {\bf Placebo Effect} {\bf A)} Population coefficient estimates for intercept, drug effect, and placebo effect for transformed phenotypes. {\bf B)} Percent improvement of phenotypes from placebo treatment and magnitude of placebo effect proportional to magnitude of drug effect. }
  \label{FIG_3}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% 4-SINGLE MEASURE %%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{Limitations of Single Measurements}

Furthermore, we found a great deal of within patient variation over repeated measurements before and after treatment. After fitting a multiple linear regression model for each patient that included the effect of drug, placebo (if applicable), and week, we calculated the standard deviation of the residuals for each patient for each phenotype (Fig \ref{FIG_4}A). \textcolor{blue}{The values for DAS ranged from near 0 to \textgreater1.5 with the mean standard deviation of within patient residuals (for DAS) was \textasciitilde0.6 points, suggesting that using a single measurement in an association study could yield different results, depending which time point is used.} \textcolor{red}{\st{ Furthermore, ranking the patients from most to least improved based on their first measurement after treatment with golimumab demonstrates the volatility of using a single measurement as a response phenotype} (Fig \ref{FIG_4}B).} The correlation between measurements in the population at any two time points after initial golimumab treatment ranges anywhere from 0.28 to 0.76 within a given phenotype (Fig \ref{FIG_4}C).

%% FIGURE 4 - SINGLE MEASURE LIMITATIONS
\begin{figure}[h!]
  \centering
  \includegraphics[width=0.9\textwidth]{Figs/4_Full.png}
  \caption{ {\bf Single Measure Limitations} {\bf A)} Distribution of within patient variance of residuals of regression model for transformed phenotypes. \textcolor{red}{\st{{\bf B)} Rank of patient improvement, sorted by least (black) to most (white) improved at 4WAG, for various phenotypes.}} {\bf C)} Pairwise Pearson correlation between phenotypes at different times after initial golimumab treatment. }
  \label{FIG_4}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% 5-DERIVED PHENOS %%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{Derived Phenotypes}

In order to make use of the repeated measurements for each individual over the course of the trial, we derived several statistics that represent the average response of a patient, the variance, or the trajectory over time. These include a simple change in the means before and after treatment, weighted to accommodate non-regular intervals between measurements (MNa), Cohen’s D statistic for measurements before and after treatment (MNcd), percent change from baseline (PRC), disease trajectory after treatment estimated by the beta coefficient of the week in a simple linear regression (Bwk), and variance of the residuals when fitting Bwk (VARwk). These statistics are correlated to varying degrees and each represents a different aspect of a patient’s response profile (Fig 5A). We began by testing these statistics against the parametric assumptions of normality and homoscedasticity. We found that estimates of post-treatment trajectory (Bwk), Cohen’s D-statistic (MNcd), and post-treatment variance (VARwk) statistics violated the assumption while the percent change from baseline and the mean change when regressed against the mean pre-treatment metrics generally adhered to these assumptions (Fig 5B). When testing the assumption of homoscedasticity for the mean change after treatment, we found that using lCRP as a phenotype conforms to this assumption while the other measurements violate it, most egregiously by rSJC and rTJC (Fig 5C). These results suggest that, even with derived statistics, it may be necessary to utilize a permutation-based method for assessing the significance of association results, depending what aspect of response one is investigating.

%% FIGURE 5 - DERIVED PHENOTYPES
\begin{figure}[h!]
  \centering
  \includegraphics[width=0.9\textwidth]{Figs/5_Full.png}
  \caption{ {\bf Derived Phenotypes} {\bf A)} Pairwise correlation between derived phenotypes. Color of columns represents phenotypic measurement. Color of rows represents derivation technique (Cyan=\% Improvement ; Purple=Mean Difference ; Orange=Cohen's D Statistic ; Gold=Trajectory after Treatment ; Pink=Within patient variance around regression fit ) {\bf B)} Shapiro-Wilk test p-values for derived statistics or residuals (versus initial disease state). {\bf C)} Breusch-Pagan test of homoscedasticity for mean change (versus initial disease state). \textcolor{orange}{Move to Supplemental Figures}}
  \label{FIG_5}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% 6-HERITABILITY %%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{Heritability Estimates}

Finally, we used the Genome-wide Complex Trait Analysis (GCTA v1.24.4) tool to estimate the heritability treatment response using several phenotypes with single time point measurements and several derived measures. SNPs with MAF of \textless1\% in our cohort were filtered out and initial disease state was used as a covariate for each phenotype when estimating heritability. The results demonstrate how several of the aforementioned issues can confound an association study. First, we showed the inconsistency of using response measurements from a single time point. At 4, 12, 20, or 28 weeks after treatment, the heritability estimate for the change in DAS is not significantly different from 0; however, when we use the difference between the final and initial measurements (weeks 100 and 0), the resulting heritability estimate is greater than 0. Similarly, for transformed SJC and TJC, the heritability estimate was significantly greater than 0 for only 1 of 5 and 3 of 5 time points used, respectively (Fig 6A). When we average the phenotypic measurements over time, we find that average change in DAS, SJC, and TJC all have heritability estimates significantly greater than 0 (Fig 6B). The transformed CRP estimate was non-significant for all 5 single measurement estimates and for the averaged phenotype. Second, we find that the heritability estimates for the transformed data are greater than the raw data and, for TJC, the transformation results in a significantly non-zero estimate. These results suggest that using a phenotype that is averaged over many measurements and transforming the data to conform to the statistical methods being implemented yield more robust measurements and are better suited for genetic association studies.

When considering the transformed results averaged over many repeated measurements, our results are consistent with previous work. The change in joint counts (rSJC, rTJC) has the greatest heritability estimate, changes in inflammation markers (lCRP or ESR) is the least heritable, and DAS (as composite score that includes the other metrics) lies in the middle. However, one must consider that rSJC and rTJC are also most susceptible to be influenced by placebo and egregiously violate some parametric assumptions of the analysis methods. Of our alternative derived phenotypes, the percent change after treatment (PRC) had the highest heritability estimates while Coehen’s D statistic (MNcd) and post-treatment disease trajectory (Bwk) had the lowest. The variance of disease state after treatment (VARwk) resulted in significant heritability estimates for SJC, TJC, and DAS, suggesting that there may be some mechanism affecting the variability of one’s response to anti-TNF agents. \textcolor{green}{SUPPLEMENTAL PLOT}

%% FIGURE 6 - HERITABILITY ESTIMATES
\begin{figure}[h!]
  \centering
  \includegraphics[width=0.9\textwidth]{Figs/6_Full.png}
  \caption{ {\bf Heritability Estimates} {\bf A)} GCTA Heritability Estimates using difference between single measurements. {\bf B)} GCTA Heritability Estimates using mean difference in disease state after treatment. (* indicates p<0.05) }
  \label{FIG_6}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% 7-PERMUTATION %%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{Permutation Testing}

As a final step, we ran permutations on several phenotypes to validate our results. We resampled the clinical data, estimated the heritability of the permuted values 1000 times for each derived phenotype, and used the likelihood ratio test statistic build a null distribution and calculate a significance level for our true data. Overall, significance levels were consistent with the p-value obtained from GCTA (Fig 7) and levels of significance remained greater for transformed phenotypes over their raw counterparts.

%% FIGURE 7 - DERIVED PHENOTYPES
\begin{figure}[h!]
  \centering
  \includegraphics[width=0.9\textwidth]{Figs/7_Full.png}
  \caption{ {\bf Derived Phenotypes} {\bf A)} Parametric vs permuted p-values for phenotypes using single time point differences. {\bf B)} Parametric vs permuted p-values for derived phenotypes from repeated measures. }
  \label{FIG_7}
\end{figure}